backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/inoutcandidate/api/IShipmentSchedulePA.java

```diff
@@ -53,6 +50,8 @@
 	@Nullable
 	ShipmentScheduleId getShipmentScheduleIdByOrderLineId(OrderLineId orderLineId);
 
+	boolean existsExportedShipmentForOrder(@NonNull final OrderId orderId);
```

Comment:
```suggestion
	boolean existsExportedShipmentScheduleForOrder(@NonNull final OrderId orderId);
```

Shipment != ShipmentSchedule